app/controllers/api/reputation_controller.rb

```diff
@@ -9,7 +9,13 @@ def index
 
       render json: SerializePaginatedCollection.(
         tokens,
-        data: tokens.map(&:rendering_data)
+        data: tokens.map(&:rendering_data),
+        meta: {
+          links: {
+            tokens: Exercism::Routes.reputation_journey_url
+          },
+          reputation_sum: current_user.reputation
```

Comment:
We've used the "total" string to describe the total amount of things. Maybe do:

```suggestion
          total_reputation: current_user.reputation
```